### ISSUE TITLE
Pull requests help

### ISSUE THREAD

**Issue by niklas2233** [Pheeeeenom/fuckigg#15]
Hey can you make a template to adding support for games so it's easier for everyone to add games to the list as it seems you haven't added a lot of games?

Or if you can make it so you can do it to every game but still have the list in case the game you want to do isnt on the list

**Comment by Pheeeeenom** [comment]
Yeah, can do that. There isn't exactly going to be much of a template as each game would be different. I can give some guidance/instructions.

**Comment by Pheeeeenom** [comment]
{
  "Moekuri: Adorable + Tactical SRPG": {
    "Exec_name": "moekurii.exe",
    "Offset": "0x2B9",
    "Patches": "0x3A"
  }
  **New game here.**
}

Generally, I would load the exe for the game in IDA Pro or any other disassembler, find the branch and the JMP/JNZ instruction and patch it. Get the offset, and follow the example above. Their DRM is basically nonexistent. If you have a game you'd like me to do as an example, I can.

**Comment by DerPolo26** [comment]
Hey, I am not really into that stuff and just found out about this here. Could you make a tutorial since you said you could do that? I'd be really happy about that.
